gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #177: Media/Static File Permanence

### Description

Fix the media/static file permanence problem.

### Testing Directions

```
./setup
python manage.py runserver
```

We are hosting this branch on https://showup-nyc-media.herokuapp.com/ to demonstrate that these changes work.

1.  Navigate to https://showup-nyc-media.herokuapp.com/.
1.  Log in as `vm1564@nyu.edu`. I uploaded a profile photo at 12/09/2019 @ 09:40pm. See that it's still there.
1.  Log in as `vm1564+1@nyu.edu`. I uploaded a profile photo at 12/09/2019 @ 09:40pm. See that it's still there.

### Miscellaneous Comments

*   @jawooson and I verified that that photos are uploaded to the AWS S3 bucket.
*   Sometimes the Travis CI - Branch build fails even though the Travis CI - Pull Request build passes. I think it has something with how fast they're running the builds so if your build ever fails in the future just restart the build and it might resolve the issue.

### Checklist
